### ISSUE TITLE
[PROPOSAL] We should DEPR this repo and stop giving support

### ISSUE THREAD

**Issue by felipemontoya** [eduNEXT/tvm#80]
I have asked in the edunext team and to some members of the communtity from Opencraft and Raccoongang to see if they use TVM and the response has been very timid. Most people know of its existence and the problem it solves, but they have found ways to develop without using this.

I think moving forward we should document those other alternative ways, write a note in the readme of this repo and stop the maintenance.

The alternatives I have seen:

1. developers that are only running in main and release, use tutor directly not requiring any extras
2. creating a virtualenv, installing tutor (any version) in it, and running command like so "TUTOR_ROOT="$(pwd)" tutor dev start"
3. using devbox with a config such as:
```{
  "packages": [
    "python@3.12",
    "nodejs@18"
  ],
  "env": {
    "TUTOR_PLUGINS_ROOT": "./plugins",
    "TUTOR_ROOT": "./root"
  },
  "shell": {
    "init_hook": [
      ". $VENV_DIR/bin/activate"
    ],
    "scripts": {
      "requirements": [
        "pip install -U -r requirements.txt"
      ]
    }
  }
}```

**Comment by mariajgrimaldi** [comment]
I agree with this proposal. Although TVM solves a relevant problem when moving between environments back and forth, there are more straightforward ways to solve it with less maintenance effort and cognitive load. I, for example, use [direnv](https://direnv.net/) with a first-time setup function written in my .zsh. 

```
function setdirenv() {
    echo "export TUTOR_ROOT=$(pwd)\nexport TUTOR_PLUGINS_ROOT=$(pwd)/plugins\nexport VIRTUAL_ENV=$(pwd)/venv\nexport PATH=$(pwd)/venv/bin:$PATH" > .envrc
    direnv allow
}
```
I haven't needed much more. However, it's worth mentioning that I mainly use tutor nightly.

**Comment by Ian2012** [comment]
I agree. I use zsh with the sintax highlight plugin with:


`source venv/bin/activate; export TUTOR_ROOT=$(pwd); export TUTOR_PLUGINS=$(pwd)/plugins`

So, whenever I load a virtual env the variables are exported. Same thing if you just press Ctrl + R and search it in the history

**Comment by MaferMazu** [comment]
I use tvm daily, but it is because it exists and is easy, and I made that a habit. However, maintaining this tool is expensive and probably not worth it.
 
If we document the alternatives, I agree to proceed with this DEPR. 👍
